src/workloads/streams/LargeHoppingWindow.yml

```diff
@@ -0,0 +1,183 @@
+        streams_startStreamProcessor: ""
+        name: *StreamProcessorName
+        pipeline: [
+          {
+            $source: {
+              connectionName: "__testMemory",
+              timeField: { $convert: { input: "$dateTime", to: "date" }},
+              allowedLateness: { size: 1, unit: "second" },
+            }
+          },
+          {
+            $hoppingWindow: {
+              interval: { size: 1, unit: "second" },
+              hopSize: { size: 250, unit: "ms" },
+              pipeline: [
+                { 
+                  $group: {
+                    _id: {
+                      auction: "$auction",
+                      url: "$url"
+                    },
+                    minPrice: { $min: "$price" },
+                    maxPrice: { $max: "$price" },
+                    sumPrice: { $sum: "$price" },
+                    avgPrice: { $avg: "$price" }
+                  }
+                }
+              ]
+            }
+          },
+          { $emit: { connectionName: "__noopSink" } }
+        ]
+        connections: [{ name: "__testMemory", type: "in_memory", options: {} }]
+  - Phase: 1..2
+    Nop: true
+  - Phase: 3
+    Repeat: 1
+    Database: *DatabaseName
+    Operations:
+    - OperationMetricsName: Stop
+      OperationName: RunCommand
+      OperationCommand:
+        streams_stopStreamProcessor: ""
+        name: *StreamProcessorName
+
+- Name: Insert_Batch1000x
+  Type: RunCommand
+  ClientName: Stream
+  Threads: *NumThreads
+  Phases:
+  - Phase: 0
+    Nop: true
+  - Phase: 1
+    Repeat: *NumBatch1000xPerThread
+    Database: *DatabaseName
+    Operations:
+    - OperationMetricsName: Insert
+      OperationName: RunCommand
+      OperationCommand:
+        streams_testOnlyInsert: ""
+        name: *StreamProcessorName
+        documents: *Batch1000x
+  - Phase: 2
+    Repeat: 1
+    Database: *DatabaseName
+    Operations:
+    - OperationMetricsName: Insert_Flush
+      OperationName: RunCommand
+      OperationCommand:
+        streams_testOnlyInsert: ""
+        name: *StreamProcessorName
+        documents:
+        - *FlushDocument
+  - Phase: 3
+    Nop: true
+
+- Name: LargeHoppingWindow.MemorySource.Stats.Insert
+  Type: StreamStatsReporter
+  ClientName: Stream
+  Database: *DatabaseName
+  Threads: 1
+  Phases:
+  - Phase: 0
+    Nop: true
+  - Phase: 1
+    Repeat: 1
+    StreamProcessorName: *StreamProcessorName
+    ExpectedDocumentCount: *ExpectedDocumentCount
+  - Phase: 2..3
+    Nop: true
+
+- Name: LargeHoppingWindow.MemorySource.Stats.Flush
+  Type: StreamStatsReporter
+  ClientName: Stream
+  Database: *DatabaseName
+  Threads: 1
+  Phases:
+  - Phase: 0..1
+    Nop: true
+  - Phase: 2
+    Repeat: 1
+    StreamProcessorName: *StreamProcessorName
+    ExpectedDocumentCount: *ExpectedDocumentCountAfterFlush
+  - Phase: 3
+    Nop: true
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - streams
+    branch_name:
+      $gte: v7.2
```

Comment:
Will this suite run continuously against new builds from master branch?

Reply:
yup! -- so good callout b/c im actually just going to rip out the `AutoRun` section, and tie this into a `test_control` bundle in DSI and update `etc/system_perf.yml`

DSI https://github.com/10gen/dsi/pull/1426
Mongo system_perf.yml https://github.com/10gen/mongo/commit/9a1efc7ea1bee73782de352ff5c598f4f564e9b5

but yeah it'll run against master branch, but how often it runs though will be based on the cron job settings in `etc/system_perf.yml` which im setting as below for now (since thats the pattern we seem to be following for all the other perf test runs.
```
cron: "0 0 * * 2,4,6"  # Tuesday, Thursday and Saturday at 00:00
```